test/renderers/boolean.field.test.tsx

```diff
@@ -420,12 +372,12 @@ test('update with undefined ref', t => {
   const input = findRenderedDOMElementWithTag(tree, 'input') as HTMLInputElement;
   t.is(input.checked, true);
 });
-
+// TODO: does it make sense to hide the input?
```

Comment:
Probably not, if the wrapping input control already takes care of this. I can't image a use case where we'd want to show the label, but not the field. This would allow us to move the test and remove the hidden prop from the field renderers